Containerfile

```diff
@@ -1,5 +1,5 @@
 FROM registry.fedoraproject.org/fedora:39 AS builder
-RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/bib
+RUN dnf install -y git-core golang gpgme-devel libassuan-devel btrfs-progs-devel lvm2-devel && mkdir -p /build/bib
```

Comment:
Looks like you ran into https://github.com/osbuild/bootc-image-builder/pull/126

Reply:
I reverted this now that your PR has landed